src/client/src/apps/MainRoute/widgets/blotter/components/blotterUtils.ts

```diff
@@ -1,10 +1,18 @@
 import { ColDef, CsvExportParams, ProcessCellForExportParams } from 'ag-grid-community'
+import { timeFormat, utcFormat } from 'd3-time-format'
 import { Trade, TradeStatus } from 'rt-types'
-import { formatDate, UtcFormatDate } from '../../spotTile/components/notional/utils'
 import SetFilter from './filters/SetFilter'
 import numeral from 'numeral'
 import { capitalize } from 'lodash'
 
+function UtcFormatDate(date: Date, format: string = '%b %e, %H:%M:%S') {
+  return utcFormat(format)(date)
+}
+
+function formatDate(date: Date, format: string = '%b %e, %H:%M:%S') {
```

Comment:
I prefer `function` myself too where it makes sense. I don't think there'll be a lint fix for it because arrows are semantically different from functions so it would be dangerous to auto change